.concourse.yml

```diff
@@ -200,6 +200,40 @@ jobs:
             context: unit-tests
             status: failure
 
+      - task: binary-test-pr
+        timeout: 40m
+        image: env-glibc
+        config:
+          platform: linux
+          inputs:
+          - name: git-pull-request-resource
+          run:
+            # user: root
+            path: sh
+            args:
+            - -exc
+            - |
+              export CARGO_HOME="$(pwd)/../cargo"
+              sudo chown $(whoami): -Rf ${CARGO_HOME} .
+              rustc +stable --version --verbose
+              cargo +stable r --bin signal-handler-test; (($? == 130))
```

Comment:
We could spawn the process within the binary, send a signal and assert the exit code, right? That would simplify running it locally as well.